pipeline/errors/error_redirect_test.go

```diff
@@ -50,6 +51,16 @@ func TestErrorRedirect(t *testing.T) {
 					assert.Equal(t, "http://test/test", rw.Header().Get("Location"))
 				},
 			},
+			{
+				d:          "should redirect with return_to param",
+				givenError: &herodot.ErrNotFound,
+				config:     `{"to":"http://test/signin","url_param":"return_to"}`,
+				assert: func(t *testing.T, rw *httptest.ResponseRecorder) {
+					assert.Equal(t, 302, rw.Code)
+					location, _ := url.Parse(rw.Header().Get("Location"))
```

Comment:
You should not discard errors. Using `require` instead of `assert` here as the test should immediately fail on error.
```suggestion
					location, err := url.Parse(rw.Header().Get("Location"))
					require.NoErr(t, err)
```